Microsoft.Toolkit.Uwp.SampleApp/Controls/SampleAppMarkdownRenderer.cs

```diff
@@ -0,0 +1,302 @@
+                        }
+                    }
+
+                    // Apply special formatting context.
+                    if (noteType != null)
+                    {
+                        if (localContext?.Clone() is UIElementCollectionRenderContext newcontext)
+                        {
+                            localContext = newcontext;
+
+                            localContext.TrimLeadingWhitespace = true;
+                            QuoteForeground = Foreground;
+                            LinkForeground = localforeground;
+                        }
+                    }
+                }
+            }
+
+            // Begins the standard rendering.
+            base.RenderQuote(element, localContext);
+
+            // Add styling to render if DocFX note.
+            if (noteType != null)
+            {
+                // Restore original formatting properties.
+                QuoteForeground = originalQuoteForeground;
+                LinkForeground = originalLinkForeground;
+
+                if (localContext == null || collection?.Any() != true)
+                {
+                    return;
+                }
+
+                // Gets the current Quote Block UI from the UI Collection, and then styles it. Adds a header.
+                if (collection.Last() is Border border)
+                {
+                    border.CornerRadius = new CornerRadius(6);
+                    border.BorderThickness = new Thickness(0);
+                    border.Padding = new Thickness(20);
+                    border.Margin = new Thickness(0, 5, 0, 5);
+                    border.Background = localbackground;
+
+                    var headerPanel = new StackPanel
+                    {
+                        Orientation = Orientation.Horizontal,
+                        Margin = new Thickness(0, 0, 0, 10)
+                    };
+
+                    headerPanel.Children.Add(new TextBlock
+                    {
+                        FontSize = 18,
+                        Foreground = localforeground,
+                        Text = symbolglyph,
+                        FontFamily = new FontFamily("Segoe MDL2 Assets"),
+                    });
+
+                    headerPanel.Children.Add(new TextBlock
+                    {
+                        FontSize = 16,
+                        Foreground = localforeground,
+                        Margin = new Thickness(5, 0, 0, 0),
+                        Text = header,
+                        VerticalAlignment = VerticalAlignment.Center,
+                        TextLineBounds = TextLineBounds.Tight,
+                        FontWeight = FontWeights.SemiBold
+                    });
+
+                    if (border.Child is StackPanel panel)
+                    {
+                        panel.Children.Insert(0, headerPanel);
+                    }
+                }
+            }
+        }
+
+        private const string NoteGlyph = "\uE946";
```

Comment:
Should this be configurable?  Or is it part of the Common Mark spec?

Reply:
This is only for the Sample App, it is part of DocFX, so it isn't included in our CommonMark-ish parser.